fc/code_generation.py

```diff
@@ -131,7 +135,39 @@ def uname(name):
     return symbols
 
 
-def create_weblab_model(path, class_name, model, outputs, parameters):
+_ONTOLOGY = None
+
+
+def get_variables_transitively(model, term):
```

Comment:
This could do with a docstring!
It's not obvious to me what this method does from the names (what transitive about it?)

Reply:
Like I said, it's hacky :) I'll move things around and document before asking for a re-review.